packages/ng/package.json

```diff
@@ -6,6 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/LuccaSA/lucca-front.git"
   },
+  "private": true,
```

Comment:
Why Private ? Lucca-Front isn't supposed to be public ?

Reply:
its not this package.json that gets published, this one serves as an entry point to build /librairies, then we publish whats inside /dist

Reply:
run `yarn build` then look whats inside `/dist`

Reply:
@jefBinomed ??

Reply:
nothing more ;) I didn't reopen the PR during the whole day